tests/sentry/backup/test_imports.py

```diff
@@ -739,6 +741,19 @@ def test_global_import_scoping(self):
             == RegionImportChunk.objects.values("import_uuid").first()
         )
 
+    def test_global_import_initial_model_deletion(self):
+        create_default_projects()
+
+        file_path = get_fixture_path("backup", "fresh-install.json")
+        with open(file_path, "rb") as tmp_file:
+            import_in_global_scope(tmp_file, printer=NOOP_PRINTER)
+
+        assert Project.objects.count() == 1
+        assert ProjectKey.objects.count() == 1
+        assert Organization.objects.count() == 1
+        assert OrganizationMember.objects.count() == 2
+        assert Team.objects.count() == 1
+
```

Comment:
Another good test: for each of the other three scopes in this test class (users, orgs, config), make add an existing model, and make sure it does NOT get wiped during the import. We want this to only wipe stuff on global imports, not any other kind (if that happens, we'll lose prod data on SaaS, which is ofc very bad!)

Reply:
done